docs/rackhd/event_notification.rst

```diff
@@ -299,4 +303,50 @@ An example of multiple filters
 Web Hook APIs
 ^^^^^^^^^^^^^^^
 
-**TODO** (The APIs of web hooks developments are ongoing, this section wiil be updated when they are finished).
+
+**Create a new hook**
+
+
+.. code-block:: REST
+
+    POST /api/2.0/hooks
+    {
+        "url": "http://www.abc.com/def"
```

Comment:
Meanwhile, suggesting to add a note here, if the `url` is already been used by any existing hooks, the POST request will fail, and return the xxxx error code (I don't remember the exact HTTP status code ^^)